dashboard/config/levels/custom/music/music-jam-functions-intro.level

```diff
@@ -28,13 +28,13 @@
             },
             {
               "type": "procedures_defnoreturn",
-              "id": "GCjIiZT^[(2+,Bh6xRa8",
+              "id": "intro_call",
```

Comment:
What's the process for authoring these? Does a levelbuilder need to go into the JSON editor and manually edit the IDs? If so might be worth capturing that info somewhere on the level edit page maybe?

Also should this be `intro-call` to match the format elsewhere?

Reply:
@mikeharv thought we might be able to add editing UI for the block ID in levelbuilder.  Separately, he suggested using the procedure ID, and might have thoughts on what might be a good UI for that as well...